people/yumao/page.en.md

```diff
@@ -0,0 +1,152 @@
+
+<PhotoScroll photos={[
+    '${path}/photos/photo1.jpg',
+]} />
+
+## After Her Leaving
+
+In Zhihu question [What are your thoughts about ZhiHu user @yu-zhang-31-35 ?](https://www.zhihu.com/question/284818437), many friends wrote their memories of yumao, and expressed their sadness and regret of the death of her.
+They also felt regrettable for Neko's death due to close relationship between Neko and Yumao.
+
+In fact, Neko didn't pass away. Before her "death", in her email in 19th March, they still encouraged each other with the words "Must stay alive".
+
+Transgender community investigated this thing once, and found the fact that Neko was pronounced dead by her family.
+
+<details>
+<summary>**Fact**</summary>
+
+Neko and Yumao were forced to separate in Shanghai. After that, Neko was brought to Kunming, Yunnan, and then hospitalized at a mental health centre for treatment.
+
+During the time, Neko always hoped to meet with Yumao. She had dinner with her parents and got an interim leave from the hospital, then she tried to contact Yumao in this time gap. After their contact, she persuaded the doctor on duty to issue discharge certificate for her.
+This action irritated her family. Shortly afterward, her family drove her to Kunming Anning Haohaizi School (hereinafter referred to as "Haohaizi") in the name of taking her to Changshui Airport.
+After that, Neko was locked in the Room 308, which can be seen as a black room.
+
+Days after she came out of the Room 308, owing to her excellent command of computer and their trust in her, the propaganda department of Haohaizi found her and asked her to fix network system. Neko accepted and fixed the system quickly, then sent a help email to Yumao at once.
+
+After that, the outside world started to lash out her parents.
+The investigative journalist of *Beijing News* also interviewed the staff of Haohaizi to ask the whereabouts of Neko.
+
+However, the parents made a shock decision.
+They said that they had already brought her out of the school, then cancelled the census register of Neko.
+In the interview of *Beijing News* , they spoke in a tearful voice and claimed that Neko had taken her own life in the early hours of 24th March.
+
+Neko was continued to be in Haohaizi since then. She was informed of Yumao's death in October. This news almost devastated her.
+
+But nevertheless, Neko still implemented the words "Must stay alive".
+She never gave up and tried to calm herself. She kept a diary and developed her hobbies and interests at that time.
+She also found some ways which could resolve her difficulties to take her mind off.
+
+During the latter time in Haohaizi, those around her helped her a lot, her friends accompanied her, and some people delivered things from the outside.
+Some friendly drillmasters would take her out to play, or bought things to her. This might make her happier.
```

Comment:
Some friendly drillmasters would take her out to play, or bought things to her. This might have made her happier.